File: Client/src/app/_models/employee.ts
import { Component } from '@angular/core';

export class Employee {
  constructor(){
    this.dateOfBirth = new Date(1985,6,6);
  }
  id: number;
  firstName: string;
  lastName: string;
  age: number;
  gender: string;
  dateOfBirth: Date;
  phoneNumber: string;
  email: string;
  city: string;
  country: string;
  isDeleted: boolean;
}

File: Server/Models/Employee.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Server.Models
{
    public class Employee
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public int Age { get; set; }
        public int YearOfJoining { get; set; }
        public string Gender { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime ModifiedDate { get; set; }
        public bool IsDeleted { get; set; }
    }
}